sdks/python/apache_beam/runners/direct/evaluation_context.py

```diff
@@ -138,7 +180,9 @@ def _pvalue_to_value(self, view, values):
     Raises:
       ValueError: If values cannot be converted into the requested form.
     """
-    return sideinputs.SideInputMap(type(view), view._view_options(), values)
+    return sideinputs.SideInputMap(type(side_input),
```

Comment:
args mentions that side_input is _UnpickledSideInput object. If that is known why do we need to do type(side_Input) here? Could the type be not _UnpickledSideInput?

Reply:
That method was existing code. I simply changed the names of a variable that appears to be an _UnpickledSideInput in the uses I gave it (and currently given by the project). But perhaps its designer intended a more generic scope. I can't tell.